libs/events/src/types/toastMessages.ts

```diff
@@ -12,7 +12,7 @@ export interface ToastMessagePayloads {
   }
 
   [ToastMessageType.SWAP_POSTED_API]: {
-    orderUid: string
+    orderId: string
```

Comment:
Any reason why here is without `U` while here is has it?
![image](https://github.com/cowprotocol/cowswap/assets/43217/1dea7399-0299-446c-9fa8-7ba88152d4dd)

Reply:
I saw in a future PR and changed it. always `u` :)